app/components/CardBet.tsx

```diff
@@ -0,0 +1,92 @@
+"use client";
+
+import { Card } from "@/app/components/ui";
+import { FixedProductMarketMaker } from "@/queries/omen";
+import { remainingTime } from "@/utils/dates";
+import Link from "next/link";
+import { Button, Logo, Tag } from "swapr-ui";
+
+interface CardBetProps {
+  market: FixedProductMarketMaker;
+}
+
+export const CardBet = ({ market }: CardBetProps) => {
+  const closingDate = new Date(+market.openingTimestamp * 1000);
+
+  return (
+    <Card className="w-full">
+      <Link key={market.id} href={`markets/${market.id}`} className="block">
+        <section className="p-4 h-[144px] flex flex-col justify-between space-y-4">
+          <div className="flex justify-between items-center">
+            <div className="flex space-x-2">
+              <Tag colorScheme="quaternary" size="sm">
+                Election
+              </Tag>
+              <Tag colorScheme="success" size="sm">
+                You choose Yes
```

Comment:
Just to have a good example, not necessary at this stage but I prefer to have it right from the start.
We should use the past tense here.

```suggestion
                You chose Yes
```